pymilvus/orm/connections.py

```diff
@@ -140,6 +140,10 @@ def connect(self, alias=DefaultConfig.DEFAULT_USING, user="", password="", **kwa
             * *password* (``str``) --
                 Optional and required when user is provided. The password corresponding to
                 the user.
+            * *secure* (``bool``) --
+                Optional. Default is false. If set to true, tls will be enabled.
+            * *cert_path* (``str``) --
+                Optional. If use a self-signed certificate, need to write the server.crt path.
```

Comment:
Will it need to specify ca.crt if CA exists?

Reply:
I run the example.py is not need to specify ca.crt.

Reply:
You should also test the case with CA.

Reply:
i should build a ca server? I don't know how to test the case with CA.